src/main/scala/cromwell/engine/db/slick/SymbolComponent.scala

```diff
@@ -100,6 +100,18 @@ trait SymbolComponent {
       if workflowExecution.workflowExecutionUuid === workflowExecutionUuid
     } yield symbol)
 
+  def symbolsFilterByWorkflowAndScopeAndNameAndIndex(workflowExecutionId: Int, scope: String, name: String, index: Int) = {
```

Comment:
I was copying from the other places where updates follow this pattern. Probably from @kshakir. Originally I had a Compiled version and Cromwell didn't compile which was when I went searching for this version